modules/openapi-generator/src/main/resources/typescript-axios/apiInner.mustache

```diff
@@ -117,7 +127,12 @@ export const {{classname}}AxiosParamCreator = function (configuration?: Configur
     {{#headerParams}}
             {{#isArray}}
             if ({{paramName}}) {
+                {{#uniqueItems}}
+                let mapped = Array.from({{paramName}}).map(value => (<any>"{{{dataType}}}" !== "Set<string>") ? JSON.stringify(value) : (value || ""));
```

Comment:
I'm not 100% sure about the comparison here `<any>"{{{dataType}}}" !== "Set<string>"` as I'm not sure what the comparison in the `Array` case is supposed to do. If someone can verify that this is correct or let me know what the correct version is supposed to be, I would be grateful 😃

Reply:
it looks correct, I think the idea is to serialize the items of a non-string array/set